product_docs/docs/epas/17/upgrading/major_upgrade/05_pg_upgrade_troubleshooting.mdx

```diff
@@ -32,3 +32,6 @@ If the original EDB Postgres Advanced Server cluster included libraries that are
 4.  Resume the upgrade process. Invoke `pg_upgrade` to perform consistency checking.
 
 4.  After you resolve any remaining problems noted in the consistency checks, invoke `pg_upgrade` to perform the data migration from the old cluster to the new cluster.
+
+!!! Note
+    Since adminpack extension is removed in this version without any alternative. Please remove it in the previous version before upgrade, instead of performing above instructions.
```

Comment:
```suggestion
    The adminpack extension has been removed in this version with no replacement. It must be removed in the previous version prior to the upgrade, instead of applying the steps described above.
```